eth/rollup.go

```diff
@@ -816,7 +816,7 @@ func (c *RollupClient) RollupEventsByBlock(blockNum int64,
 			L1Tx.UserOrigin = true
 			L1Tx.EthTxHash = vLog.TxHash
 			//Get l1Fee in eth wei spent in the l1 tx
-			tx, _, err := c.client.client.TransactionByHash(context.Background(), vLog.TxHash)
+			tx, _, err := c.ethereumClient.client.TransactionByHash(context.Background(), vLog.TxHash)
```

Comment:
I think in here we need to take this PR opportunity to update this old code smell too.
c = client
ethereumClient
and.....  another client.
It does not look good.